src/gui/dialogues/newpilotdialog.h

```diff
@@ -49,6 +49,8 @@ private slots:
 
     Pilot oldEntry;
 
+    QPair<QString, int> oldPosition;
+
```

Comment:
this goes back to the pevious comments. Why would you have the position twice? one *in* the Entry and one *outside* of it? Leakage again i see xD

Reply:
That was a workaround because I couldn't get the PilotEnty copied into the dialog with its copy operator deleted. How would you go about that, enable the copy operator for entry or is there a better way?

Reply:
You would simply pass in the existing entry upon construction and keep a pointer to it. you only need the `oldEntry` which has position encapsulated already